sdk/src/main/java/io/opentelemetry/sdk/trace/Samplers.java

```diff
@@ -43,10 +43,22 @@
 @Immutable
 public final class Samplers {
 
-  private static final Sampler ALWAYS_ON = new AlwaysOnSampler();
-  private static final Sampler ALWAYS_OFF = new AlwaysOffSampler();
-  private static final Decision ALWAYS_ON_DECISION = new SimpleDecision(/* decision= */ true);
-  private static final Decision ALWAYS_OFF_DECISION = new SimpleDecision(/* decision= */ false);
+  private static final Decision ALWAYS_ON_DECISION =
```

Comment:
Now that this is used for something other than the always-on sampler, I think we should rename this to just `SIMPLE_ON_DECISION` or maybe even just `ON_DECISION`

Reply:
Related: #1256. Maybe EMPTY_(NOT_)SAMPLED_DECISION?

Reply:
Done.